### ISSUE TITLE
DOC,ENH: How does anaconda-project work with repo2docker (REES)?

### ISSUE THREAD

**Issue by westurner** [Anaconda-Platform/anaconda-project#267]
- https://repo2docker.readthedocs.io/en/latest/
  - https://repo2docker.readthedocs.io/en/latest/config_files.html
  - REES: Reproducible Execution Environment
    https://repo2docker.readthedocs.io/en/latest/specification.html
- "DOC: How to launch anaconda-project.yml and/or add to REES?"
  https://github.com/jupyter/repo2docker/issues/897

**Comment by jbednar** [comment]
I've wanted there to be some integration/interoperatibility there between anaconda-project and repo2docker/binder/REES, and have discussed that with Chris Holdgraf, but that idea stalled out as soon as it started due to personnel changes.

You can see https://github.com/pyviz-topics/examples for examples of setting up a project to work with binder, but there isn't an exact match, because repo2docker assumes you'd have a single project per repository, while anaconda-project is happy to have a different project in each subdirectory.  anaconda-project also specifies the command to run as part of the project description, which REES did not include last time I checked on it, so you have to do hacks like https://github.com/holoviz/jupyter-panel-proxy to launch the project if it's anything but a Jupyter notebook.

Note that anaconda-project is also getting the ability to generate Docker images by itself (https://github.com/Anaconda-Platform/anaconda-project/pull/246), so it won't necessarily need repo2docker in any case.

**Comment by AlbertDeFusco** [comment]
I've not looked at repo2docker, but here's the s2i builder for anaconda-project. I've been working on this and #246 in parallel. There is certainly an opportunity to integrate with repo2docker

https://github.com/Anaconda-Platform/s2i-anaconda-project

**Comment by westurner** [comment]
cc @choldgraf

**Comment by choldgraf** [comment]
Feel free to reach out in the repo2docker issues or jupyter discussion forums etc if there are particular questions or blockers that you all have. It would be great to see unification for reproducibility specs etc across the community!

**Comment by mcg1969** [comment]
Thanks for opening this issue, and apologies for the long delay in responding.

Anaconda Project has not been actively developed in some time and has largely been superseded by newer alternatives:

* [pixi](https://pixi.sh/) and [uv](https://docs.astral.sh/uv/) represent the first truly successful successors.
* [conda-workspaces](https://github.com/conda-incubator/conda-workspaces) is a promising pixi-compatible alternative for users who prefer to stay closer to the conda ecosystem.

Going forward, work on this repository is focused on critical bug fixes and on tooling to help users migrate existing Anaconda Project projects to pixi (and, by extension, conda-workspaces). See the [Pixi support](https://anaconda-project.readthedocs.io/en/latest/pixi-support.html) page in our documentation for details on the `anaconda-project export-pixi` command, which can convert an existing `anaconda-project.yml` into a `pixi.toml` for you.

The `anaconda-project` package itself will remain available, so existing projects relying on its stable feature set can continue to be supported. However, we do not anticipate further enhancement work, and for that reason we are closing out the bulk of our open issue backlog — including this one.

If this issue describes a critical regression in the stable feature set (as opposed to an enhancement or an edge case in a feature we no longer plan to expand) and you have a minimal reproducer against a current release, please feel free to reopen and we will take another look.

Thank you for your interest in Anaconda Project.
